src/Host/Client/Impl/Host/BrokerClient.cs

```diff
@@ -28,29 +28,24 @@ internal abstract class BrokerClient : IBrokerClient {
             TimeSpan.FromSeconds(5);
 #endif
 
-        protected DisposableBag DisposableBag { get; } = DisposableBag.Create<BrokerClient>();
-
-        private readonly LinesLog _log;
         private readonly string _interpreterId;
         private AboutHost _aboutHost;
 
+        protected DisposableBag DisposableBag { get; } = DisposableBag.Create<BrokerClient>();
```

Comment:
yeah, found it

---
In reply to: [81240024](https://github.com/Microsoft/RTVS/pull/2408#discussion_r81240024) [](ancestors = 81240024,81231658)